huff_codegen/src/irgen/arg_calls.rs

```diff
@@ -122,21 +122,31 @@ pub fn bubble_arg_call(
                         }
```

Comment:
Maybe delete the Opcode if-let on L49-55? It doesn't seem to ever be hit.

Also need to fix passing constants as args, but that can be done in another PR if you'd like :smile: